launcher/config_test.go

```diff
@@ -451,6 +452,7 @@ func TestConfigurePropagators(t *testing.T) {
 	assert.Greater(t, len(carrier.Get("x-b3-traceid")), 0)
 	assert.Contains(t, carrier.Get("otcorrelations"), "keytwo=bar1")
 	assert.Contains(t, carrier.Get("otcorrelations"), "keyone=foo1")
+	assert.Greater(t, len(carrier.Get("traceparent")), 0)
```

Comment:
Nit: can this me made more specific (I mean, test for a specific length)?

Reply:
This test is really just checking that `traceparent` is set, the validity of its content is outside the scope of this test here.